### ISSUE TITLE
`arki-check --unarchive` documentation

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/arkimet#297]
There's no mention of the `--unarchive` option in https://arpa-simc.github.io/arkimet/datasets/archive.html (or in any other part of the doc).

The only mention I found is in the help and in the man page of `arki-check`
```
   --unarchive pathname  Given a pathname relative to .archive/last, move it
                         out of the archive and back to the main dataset
```

This is a bit misleading since by trial and error it seems that it accepts only specific filenames (no paths, no wildcards).
Is this correct?

**Comment by spanezz** [comment]
Let's work out how it works first, then where to document it.

In theory, if you have somethign like `datasets/lami123/.archive/last/2022/2022-12.grib`, you can do this:

```
arki-check --unarchive 2022/2022-12.grib datasets/lami123
```

And this should move that segment into `datasets/lami123/2022/2022-12.grib`, and index it as part of the online dataset.

Does this match the behaviour you observe?

**Comment by spanezz** [comment]
Right, yes, I see I have work to do to make it not just working, but also useable.

It makes sense to make it take segment names, and infer datasets from them.

I'll work on this
